genesis/genesis_local.go

```diff
@@ -60,11 +60,15 @@ var (
 					gas.DBWrite:   1,
 					gas.Compute:   1,
 				},
-				MaxCapacity:              1_000_000,
-				MaxPerSecond:             1_000,
-				TargetPerSecond:          500,
-				MinPrice:                 1,
-				ExcessConversionConstant: 5_000,
+				MaxCapacity:     1_000_000, // Max block size ~1MB
+				MaxPerSecond:    250_000,
+				TargetPerSecond: 125_000, // Target block size ~125KB
+				MinPrice:        1,
+				// ExcessConversionConstant = (Capacity - Target) * NumberOfSecondsPerDoubling / ln(2)
```

Comment:
(1000000 - 125000) * (30 / ln(2)) is not 5.4 million.

Reply:
Sorry, was supposed to read as `Capacity (per second) - Target (per second)` (like in the SoV calculations below).

Updated to use the exact names in the config.

Reply:
(The calculation is: `(250_000 - 125_000) * (30 / ln(2))`